manifests/charts/ambient/Chart.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: v2
+name: ambient
+description: Helm umbrella chart for ambient
+type: application
+version: 1.0.0
+appVersion: 1.0.0
+kubeVersion: ">= 1.23.0-0"
```

```suggestion
kubeVersion: ">= 1.23.0-0"
```

This can be removed if it's controversial (and maybe automatically updated somehow on release), but it mimics our documented k8s requirements and is a nice safeguard.

---

Does it work with `helm template`? I don't know what "work" means exactly, I suppose that it ignores it entirely?

IIRC `helm template` will bake in some arbitrary version, wondering if they actually are using that here which would be a bit silly. Also probably fine since its presumably above 1.23, but a bit odd to effectively be accidentally doing a helm version check instead of a kube version check for that case

---

`helm template` has a `--kube-version` flag for this, but IDK what the default is